### PR TITLE
Feature users and posts

### PR DESCRIPTION
**Pull request by BogdanScherban**
1) New ReadMe file;
2) Create/Update action redirects to the list;
3) Id with random number for creation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,1623 +1,63 @@
...
+Node-RED is a flow-based programming tool for wiring together hardware devices, APIs and online services in new and interesting ways.
+                                           
+It provides a browser-based editor that makes it easy to wire together flows using the wide range of nodes in the palette that can be deployed to its runtime in a single-click.
 
-When you run `create-react-app`, it always creates the project with the latest version of `react-scripts` so you’ll get all the new features and improvements in newly created apps automatically.
+It was developed by IBM’s Emerging Technology Services team and now a part of the JS Foundation.
 
-To update an existing project to a new version of `react-scripts`, [open the changelog](https://github.com/facebookincubator/create-react-app/blob/master/CHANGELOG.md), find the version you’re currently on (check `package.json` in this folder if you’re not sure), and apply the migration instructions for the newer versions.
+Tutorial for Node-Red is available by this link:
+https://nodered.org
```
Comment (tony-shannon) [inline README.md:32]: Need to mention Redis here also please ,as data persisted in Redis which is key feature

File: README.md [hunk 2]
```diff
@@ -1,1623 +1,63 @@
...
-   res.sendFile(path.join(__dirname, './build', 'index.html'));
- });
-```
-
-If you’re using [Apache](https://httpd.apache.org/), you need to create a `.htaccess` file in the `public` folder that looks like this:
-
-```
-    Options -MultiViews
-    RewriteEngine On
-    RewriteCond %{REQUEST_FILENAME} !
```
Comment (tony-shannon) [inline README.md]: suggest cd to cd RA-NR-demo  (suggested project directory name)

Comment (tony-shannon) [inline README.md]: because server side hasnt been installed yet

Comment (tony-shannon) [inline README.md]: NodeRed console is available on http://[localhost]:1880

File: public/index.html [hunk 3]
```diff
@@ -30,7 +30,7 @@
 
     <script type="text/javascript">
       window.config = {
-        domainName: "http://localhost:1880",
+        domainName: "http://reactadmin-on-nodered.ripple.foundation:1880",
```
Comment (tony-shannon) [inline public/index.html]: this should not be hard coded like this

Comment (tony-shannon) [reply public/index.html]: You also need to explain how to fire up the application , which URL to use? its not localhost:1880

**Review by tony-shannon — CHANGES_REQUESTED** [review]
see comments that need action before merging

File: public/index.html (hunk 3 above)
Comment (BogdanScherban) [reply public/index.html]: Now I left the empty value. In ReadMe file I added info, that it is necessary to define domain name in index.html.

File: README.md (hunk 2 above)
Comment (BogdanScherban) [reply README.md]: Added

Comment (BogdanScherban) [reply README.md]: Fixed

Comment (BogdanScherban) [reply README.md]: Renamed

File: README.md (hunk 1 above)
Comment (BogdanScherban) [reply README.md:32]: Added, about NodeJS and Redis

**Comment by BogdanScherban** [comment]
@tony-shannon 
Tony, I fixed all your notes to this PR.

**Pull request merged**